<launch>
  <node pkg="obj_tf" type="broadcastObj.py" name="obj_tf_broadcaster"  output="screen">
    <param name="scale_linear" value="0.5" type="double"/>
    <param name="scale_angular" value="1.5" type="double"/>
  </node>
</launch>
